src/services/services.ts

```diff
@@ -3558,6 +3558,7 @@ function getSymbolAtLocationForQuickInfo(node: Node, checker: TypeChecker): Symb
  * @internal
  */
 export function getPropertySymbolsFromContextualType(node: ObjectLiteralElementWithName, checker: TypeChecker, contextualType: Type, unionSymbolOk: boolean): readonly Symbol[] {
+    contextualType = contextualType.getNonNullableType();

```

Comment:
This just fine-tunes the algorithm for a common case. By discarding nullable (that can't provide anything useful here as they don't have properties anyway) the `!contextualType.isUnion()` "fast path" is taken here. This allows for better symbol result to be returned from here - despite those locations having errors. Without this the symbol of the anonymous object literal type (the one with error) is picked and that's less useful.